.github/workflows/net9-deploy.yml

```
@@ -0,0 +1,69 @@
+name: TUO-Net9-Deploy
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    branches: [merge-cuo]
+    workflows: [Build-Test]
+    types:
+      - completed
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)